npm/css-src/sass/_header.scss

```diff
@@ -1,8 +1,16 @@
 .govuk-header__tna-logo {
-    width: 33.33%;
+    width: 220px;
     float: left;
 }
 
 .govuk-header__tna-logo-image {
     width: 220px;
+}
+
+.govuk-header__tna-content {
+    width: 290px;
+    position: relative;
+    align-self: center;
```

Comment:
Since 'align-self' property overrides a grid or flex item's align-items value but as we aren't using grid or flex items, it is not required here and it doesn't affect anything with regards to how the header is positioned so it has been removed.